### PR TITLE
Populate for .get method; Move prisma-adapter

### PR DESCRIPTION
**Pull request by noluckjustskill**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@
...
 | [moleculer-db-adapter-mongoose](/packages/moleculer-db-adapter-mongoose#readme) | [![NPM version](https://img.shields.io/npm/v/moleculer-db-adapter-mongoose.svg)](https://www.npmjs.com/package/moleculer-db-adapter-mongoose) | Mongoose adapter for Moleculer DB service |
 | [moleculer-db-adapter-sequelize](/packages/moleculer-db-adapter-sequelize#readme) | [![NPM version](https://img.shields.io/npm/v/moleculer-db-adapter-sequelize.svg)](https://www.npmjs.com/package/moleculer-db-adapter-sequelize) | SQL adapter (Postgres, MySQL, SQLite & MSSQL) for Moleculer DB service |
-| [moleculer-db-adapter-prisma](/packages/moleculer-db-adapter-prisma#readme) | [![NPM version](https://img.shields.io/npm/v/moleculer-db-adapter-prisma.svg)](https://www.npmjs.com/package/moleculer-db-adapter-prisma) | Prisma adapter for Moleculer DB service |
+| [moleculer-db-adapter-prisma](https://github.com/noluckjustskill/moleculer-db-adapter-prisma#readme) | [![NPM version](https://img.shields.io/npm/v/moleculer-db-adapter-prisma.svg)](https://www.npmjs.com/package/moleculer-db-adapter-prisma) | Prisma adapter for Moleculer DB service |
```
Comment (icebob) [inline README.md:21]: The readme lists only the adapter inside the repo. 3rd party adapters are in the awesome repo.

File: packages/moleculer-db/src/index.js [hunk 2]
```diff
@@ -427,13 +427,13 @@ module.exports = {
 		 * @param {Boolean?} decoding - Need to decode IDs.
 		 * @returns {Object|Array<Object>} Found entity(ies).
 		 */
-		getById(id, decoding) {
+		getById(id, populate, decoding) {
```
Comment (icebob) [inline packages/moleculer-db/src/index.js:430]: It's a breaking change, `getById` is an exposed method.

**Review by icebob — CHANGES_REQUESTED** [review]

File: packages/moleculer-db/src/index.js (hunk 2 above)
Comment (noluckjustskill) [reply packages/moleculer-db/src/index.js:430]: Yes, it is. But `populate` for "get" method doesn't work now https://moleculer.services/docs/0.13/moleculer-db.html#get

Comment (icebob) [reply packages/moleculer-db/src/index.js:431]: Here is a test which covers that populate works with the "get": https://github.com/moleculerjs/moleculer-db/blob/master/packages/moleculer-db/test/integration/populates.spec.js#L117

Populate implemented in the `transformDocuments` and not in the adapter.

**Pull request closed without merging**